src/yuzu/main.h

```diff
@@ -252,6 +255,7 @@ private slots:
     void OnPauseGame();
     void OnPauseContinueGame();
     void OnStopGame();
+    void OnPrepareForSleep(bool sleep);
```

Comment:
```suggestion
    void OnPrepareForSleep(bool prepare_sleep);
```
Missed renaming this parameter?